statsmodels/tsa/statespace/tests/results/test_sarimax_stata.do

```diff
@@ -1,5 +1,5 @@
 // Example 1: ARIMA model
-use http://www.stata-press.com/data/r12/wpi1, clear
+use https://www.stata-press.com/data/r12/wpi1, clear
```

Comment:
I need to check if Stata actually uses https or whether it redirects as needed or needs http
here they use http links
https://www.stata-press.com/data/r12/mv.html

Reply:
Stata 14 which I have on this computer, still uses `use http://...` in the documentation (docstring for `use`).
However, both http and https addresses work to download the data inside Stata interpreter.

Reply:
Stata supports https

https://www.stata-press.com/data/r12/wpi1.dta

Reply:
No redrect

![image](https://user-images.githubusercontent.com/5585221/44536038-75e4e900-a6f3-11e8-8bcc-0e9267198e21.png)